sinks/librato.py

```diff
@@ -181,6 +186,10 @@ def add_measure(self, key, value, time):
         if self.prefix:
             name = "%s.%s" % (self.prefix, name)
 
+        # Add a source prefix
+        if self.source_prefix:
+            source = "%s%s" % (self.source_prefix, source)
```

Comment:
I agree. I'm fine with requiring the dot delimiter for both and staying backwards compatible...

Sent from my iPhone

> On Sep 14, 2015, at 7:57 PM, Mike Heffner <notifications@github.com> wrote:
> 
> In sinks/librato.py:
> 
> > @@ -181,6 +186,10 @@ def add_measure(self, key, value, time):
> >          if self.prefix:
> >              name = "%s.%s" % (self.prefix, name)
> >  
> > +        # Add a source prefix
> > +        if self.source_prefix:
> > +            source = "%s%s" % (self.source_prefix, source)
> Might be a bit confusing that the metric prefix uses a period delimiter while the source_prefix does not?
> 
> —
> Reply to this email directly or view it on GitHub.
>